JustLog/Classes/Logger.swift

```diff
@@ -9,24 +9,12 @@
 import UIKit
 import SwiftyBeaver
 
-@objcMembers
-public final class Logger: NSObject {
-    
-    public enum LogType: String {
-        case debug
-        case warning
-        case verbose
-        case error
-        case info
-    }
-    
-    internal struct QueuedLog {
-        let type: LogType
-        let message: String
-        let file: String
-        let function: String
-        let line: UInt
-    }
+public protocol Logging {
+       
+    func log(with log: Loggable)
```

Comment:
I'm not sure `with` is appropriate here? We're logging the `Loggable` instance right? Do we even need a label?

Reply:
@antoniostrijdom I agree. What do you think if we remove `with` label and rename that method to 
`send(_ log: Loggable)`?
 
Example of method call
`Logger.shared.send(Log(...))`